CablePairTester_CommonCfg.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * CablePairTester_CommonCfg.c
+ *
+ *  Created on: Feb 22, 2021
+ *      Author: PUL1KOR
+ */
+
+
+#include "CablePairTester_CommonCfg.h"
+
+const int MAX_COLORPAIR_NAME_CHARS = 16;
+
+const char* MajorColorNames[] = { "White", "Red", "Black", "Yellow", "Violet" };
+const char* MinorColorNames[] = { "Blue", "Orange", "Green", "Brown", "Slate" };
+
+int numberOfMajorColors_int = sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+int numberOfMinorColors_int = sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+/**************
+ * Description : This will create string of the passed colour pairs
+ * In: ColorPair --> Structure pointer.

```

Comment:
Avoid comments that do not add more information than the code itself. Why is it needed to describe ColorPair as a Structure pointer?